src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsidentifyremandperiodsapi/relevantremand/model/Charge.kt

```diff
@@ -18,6 +18,25 @@ data class Charge(
   val isActiveBooking: Boolean = false,
   val isRecallSentence: Boolean = false,
   val isTermSentence: Boolean = false,
+  val resultCode: String?,
```

Comment:
I've tried to keep the result codes (NOMIS IDs). Out of the model. Could this just be a boolean and the result code checked in the mapping from prison-api.